ecl/hql/hqlgram.y

```diff
@@ -1741,6 +1742,14 @@ persistOpt
     : fewMany
     | expireAttr
     | clusterAttr
+    | REFRESH '(' TOK_TRUE ')'
```

Comment:
This should be REFRESH '(' expression ')'
and then code to ensure it is boolean and constant in the production.  Otherwise

   doRefresh := true;
   ...REFRESH(doRefresh);

will not be supported.